### ISSUE TITLE
[Login Widget] Missing Styling options for 'Logged in message'

### ISSUE THREAD

**Issue by VerdiH** [elementor/elementor#7928]
**What problem is your feature request going to solve? Please describe.**
No more extra CSS to change styling of the 'logged in message'-text. It's what you see when using the Login Widget after you've actually been logged in. In the Engilsh version it's this text "You are Logged in as NAME (Logout)".

**Describe the solution you'd like**
- Currently you have these in the styling tab: Form, Labels, Fields, Button
- Please consider adding "Message" to this.
- After opening that, a user gets these options: Spacing, Text Color and Typography.

**Describe alternatives you've considered**
CSS.

**Ps. If a user want to style "Remember me" it's necessary to toggle "Label" to show**
Then remove the unwanted labels before styling that text. I believe this could be more more user friendly and suggest to consider adding styling option for this one too in the options box I suggested above.

Please let me know if you need me to elaborate.

**Comment by VerdiH** [comment]
Ps. this is the very simple CSS-snippet I use to fix it for now:

```
.elementor-login {
    color: #fff;
    text-align: center;
}
.elementor-login a {
    color: #00f1ff;
}
```

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.6.0

Feel free to update

Thanks!

**Comment by Jeppebropedersen** [comment]
It would be nice if it was translatable or if it came with a customizable "logged in" message.
